internal/resources/fetcher.go

```diff
@@ -25,6 +27,9 @@ type Fetcher struct {
 	// Namespace where the Kubewarden components (e.g. policy server) are installed
 	// This is the namespace used to fetch the policy server resources
 	kubewardenNamespace string
+	// FQDN of the policy server to query. If not empty, Fetcher will query on
+	// port 3000. Useful for out-of-cluster debugging
```

Comment:
```suggestion
	// `localhost:3000`. Useful for out-of-cluster debugging
```

Reply:
Thing is, for knowing the FQDN we need to know the service of the PolicyServer pod, hence why it happens later.

We want the in-cluster behaviour as default, so that's why I left it empty, If we want our own (`localhost`, `cluster.local`, whatever the developer uses), we give it value with the flag.